gnat2goto/driver/tree_walk.adb

```diff
@@ -1178,7 +1179,7 @@ package body Tree_Walk is
          null;
       else
          if Kind (Constant_Resolved_Type) in Class_Bitvector_Type then
-            Constant_Width := Get_Width (Constant_Resolved_Type);
+            Constant_Width := Get_Width (Constant_Resolved_Type) * 4;
```

Comment:
Rather than this bit of magic arithmetic, could you add a helper function so that it has a descriptive name to indicate the purpose of the operation? e.g. `Fully_Expanded_Bitvector_Width` or something else appropriate?

Reply:
This was undone in a later commit